sapphire/tests/transformations/test_celestial.py

```diff
@@ -155,11 +171,101 @@ def test_against_jpl_horizons(self):
+    """
+    def test_pyephem(self):
+        from sapphire.transformations import celestial
+        import numpy as np
+        # This is the transform inputs
+        eq = [(-39.34633914878846, -112.2277168069694, 1295503840,
+               3.8662384455822716, -0.31222454326513827),
+              (53.13143508448587,-49.24074935964933, 985619982,
+               3.901575896592809, -0.3926720112815971),
+              (48.02031016860923, -157.4023812557098, 1126251396,
+               3.366278312183976, -1.3610394240813288)]
+        # transform inputs converted to of zenaz inputs
+        zeneq = [(-39.34633914878846, -112.2277168069694,
+                  1295503840, 1.8830208700600348, -2.295442118787375),
+                 (53.13143508448587, -49.24074935964933, 985619982,
+                  1.9634683380764937, -2.3307795697979126),
+                 (48.02031016860923, -157.4023812557098, 1126251396,
+                  2.9318357508762256, -1.7954819853890793)]
+
+        # result of pyephem hor->eq/zenaz-> eq
+        efemeq = [(5.620508199785029, -0.3651173667585858),
+                 (5.244630787139936, -0.7866376569183651),
+                 (2.276751381056623, -1.0406498066785745)]
+        # result of pyephem eq->hor
+        altaz = [(2.175107479095459, -0.19537943601608276),
+                 (5.25273323059082, -0.8308737874031067),
+                 (3.4536221027374268, -0.894329845905304)]
+        # result converted to eq->zenaz
+        zenaz = [(1.7661757628109793, -0.6043111523005624),
+                 (2.4016701141980032, 2.6012484033836625),
+                 (2.4651261727002005, -1.8828257759425302)]
+
+        htoea_test = []
+        etoha_test = []
+        zenaztoeq_test = []
+        eqtozenaz_test = []
+
+        # Produce horizontal_to_equatorial_astropy results
+        for i in eq:
+            result = celestial.horizontal_to_equatorial_astropy(
+                i[0], i[1], i[2], [(i[3], i[4])])
+            htoea_test.extend(result)
+
+        # Produce equatorial_to_horizontal_astropy results
+        for i in eq:
+            result = celestial.equatorial_to_horizontal_astropy(i[0], i[1],
+                                                                i[2],
+                                                                [(i[3], i[4])])
+            etoha_test.extend(result)
+
+        # Produce zenithazimuth_to_equatorial_astropy results
+        for i in zeneq:
+            result = celestial.zenithazimuth_to_equatorial_astropy(
+                i[0], i[1], i[2], [(i[3], i[4])])
+            zenaztoeq_test.extend(result)
+
+        # Produce equatorial_to_zenithazimuth_astropy results
+        for i in eq:
+            result = celestial.equatorial_to_zenithazimuth_astropy(i[0], i[1],
+                                                                i[2],
+                                                                [(i[3],
+                                                                  i[4])])
```

Comment:
Try this instead:
```
result = celestial.equatorial_to_zenithazimuth_astropy(
    i[0], i[1], i[2], [(i[3], i[4])])
```
except then also fix the usage of `tuple[index]`, as Tom mentioned.